src/languages/php.js

```diff
@@ -477,6 +477,7 @@ export default function(hljs) {
   };
 
   return {
+    name: 'PHP',
```

Comment:
```suggestion
    name: 'PHP (snippet)',
```

vs `template`... PHP alone is ambiguous.